pkg/cmd/cli/cmd/edit.go

```diff
@@ -23,21 +22,35 @@ import (
 	"github.com/openshift/origin/pkg/util/jsonmerge"
 )
 
+// EditOptions is a struct that contains all variables needed for cli edit command.
+//These variables are used in Complete, Validate and RunEdit methods.
+type EditOptions struct {
+	out       io.Writer
+	printer   kubectl.ResourcePrinter
+	namespace string
+	rmap      *resource.Mapper
+	args      []string
+	builder   *resource.Builder
+	cmd       *cobra.Command
```

Comment:
We shouldn't carry the Command struct inside our *Options. Remove this and use o.out instead of o.Cmd.Out() in RunEdit.